examples/tracegen/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+services:
+  # Jaeger
+  jaeger:
+    image: jaegertracing/all-in-one:latest
+    ports:
+      - "16686:16686"
+      - "14250"
+
+  otel-collector:
+    build:
+      context: ../..
+      dockerfile: examples/tracing/Dockerfile
+    command: ["--config=/etc/otel-collector-config.yml"]
+    volumes:
+      - ./otel-collector-config.yml:/etc/otel-collector-config.yml
+    ports:
+      - "1888:1888"   # pprof extension
```

Comment:
Is this relevant for the test? Keep the usage surface minimal, to avoid future compatibility issues.

Reply:
fixed.